cli/dcoscli/data/help/package.txt

```diff
@@ -8,17 +8,17 @@ Usage:
     dcos package describe <package-name>
                           [(--app [--render]) --cli --config --options=<file>]
                           [--package-version=<package-version>]
-    dcos package describe <package-name> --package-versions
+                          [--package-versions]
```

Comment:
i agree the command is confusing. are you sure this makes it clearer? to me it is even more confusing because it doesn't follow the same convention as the other commands. i think the original issue was that the command help was not alphabetized - making this similar command appear as a possible typo:
```
    dcos package describe [--app --cli --config]
                          [--render]
                          [--package-versions]
                          [--options=<file>]
                          [--package-version=<package-version>]
                          <package-name>
```

the change i was suggesting was to alphabetize, and to make consistent with the rest of the command help.